qemu.go

```diff
@@ -468,13 +470,16 @@ func (q *qemu) setCPUResources(podConfig PodConfig) ciaoQemu.SMP {
 	return smp
 }
 
-func (q *qemu) setMemoryResources(podConfig PodConfig) ciaoQemu.Memory {
+func (q *qemu) setMemoryResources(podConfig PodConfig) (ciaoQemu.Memory, error) {
+	hostMem, err := getHostMemorySize()
+	if err != nil {
+		return ciaoQemu.Memory{}, fmt.Errorf("Unable to read memory info: %s", err)
+	}
+	// add 1G memory space for nvdimm device (vm guest image)
+	memMax := fmt.Sprintf("%dM", int(float64(hostMem)/1024/1024)+1024)
```

Comment:
emm... the code itself is clear enough. Since `memMax` is a string, I think I'm gonna change `hostMem` to `hostMemBytes` to be clearer.